modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/domain/cell/L0Cell.scala

```diff
@@ -47,6 +52,8 @@ class L0Cell[F[_]: Async](
                 Algebra.enqueueDAGL1Data(l1OutputQueue)(data)
               case EnqueueUpdateNodeParameters(updateNodeParameters) =>
                 Algebra.enqueueUpdateNodeParameters(updateNodeParametersQueue)(updateNodeParameters)
+              case EnqueueDelegatedStake(data) => Algebra.enqueueDelegatedStake(delegatedStakeOutputQueue)(data)
+              case EnqueueNodeCollateral(data) => Algebra.enqueueNodeCollateral(nodeCollateralOutputQueue)(data)
```

Comment:
Same here (will skip references after this that apply to same issue, but its repeated.)